.github/workflows/multi-node-test-workflow.yml

```diff
@@ -29,20 +29,20 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
-          ref: '1.0'
+          ref: '1.x'
       - name: Build OpenSearch
         working-directory: ./OpenSearch
-        run: ./gradlew publishToMavenLocal -Dbuild.snapshot=false
+        run: ./gradlew publishToMavenLocal
       # This step adds dependency, common-utils
       - name: Checkout common-utils
         uses: actions/checkout@v2
         with:
           repository: 'opensearch-project/common-utils'
           path: common-utils
-          ref: '1.0'
```

Comment:
Are we not updating the version of common utils to 1.1 or more specifically 1.x?

Reply:
AFAIK common-utils is developing 1.1 on main right now because they aren't actively pursuing 2.0 yet. So the 1.x branch is a week behind.